static/src/js/components/Map/Map.scss

```diff
@@ -1,30 +1,30 @@
 @use '../../../css/utils';
 
-.edsc-map-spatial-drawing__cancel-wrapper {
+.edsc-map__spatial-drawing__cancel-wrapper {
```

Comment:
These classes should probably be renamed to 'map' not 'edsc-map' for consistency